curriculum/challenges/english/10-coding-interview-prep/rosetta-code/sailors-coconuts-and-a-monkey-problem.english.md

```diff
@@ -51,7 +51,6 @@ tests:
 <div id='js-seed'>
 
 ```js
-// noprotect
```

Comment:
@ojeytonwilliams @RandellDawson is loop-protect needed for these? Any idea why it was formerly disabled?

Reply:
Loop protect ignores that comment now. We can safely remove them all.

It's now done here https://github.com/freeCodeCamp/freeCodeCamp/blob/8788ad946b02e4fdbbe7dc45e5fd906aad6ab1d3/client/src/templates/Challenges/utils/build.js#L196-L198